integration-tests/funqy-google-cloud-functions/README.md

```diff
@@ -0,0 +1,77 @@
+# Google Cloud Functions - Funqy Binding
+
+This integration test has no automated test, it needs to be launched manually.
+
+## Build the artifact
+
+First, you need to log in to Google Cloud:
+
+```shell script
+gcloud auth login
+```
+
+Then you need to use Maven to build the artifact and put it in a directory where it will be the only file.
+
+```shell script
+mkdir deployment
+mvn clean package
+cp target/quarkus-integration-test-funqy-google-cloud-functions-999-SNAPSHOT-runner.jar deployment/
+```
+
+Finally, you need to use `gcloud` to deploy the function to Google Cloud. The `gcloud` command is different for HttpFunction and
+Background function so the set of instruction differs for each.
+
+This example contains multiple Funqy functions, if you want to test a different function that the one defined inside
+your `application.properties`, you can use the `--set-env-vars` option of `gcloud` to define the name of the function via the 
+`QUARKUS_FUNQY_EXPORT` environment variable.
+
+## HTTP function
+To deploy the HttpFunction, you can use the following `gcloud` command:
+
+```shell script
+gcloud beta functions deploy quarkus-funqy-http --entry-point=io.quarkus.funqy.gcp.functions.FunqyHttpFunction \
+  --trigger-http \
+  --runtime=java11 --source=deployment
+```
+
+After deploying your HTTP function to Google Cloud, the `gcloud` command will output the endpoint base location.
+
+You can issue the following `curl` commands to test it:
+
+```shell script
+curl -v {httpsTrigger.url}
+```
+
+## Background function
+
+### PubSub event
+
+To deploy a background function that listen to PubSub event, you can use the following `gcloud` command:
+
+```shell script
+gcloud beta functions deploy quarkus-funqy-pubsub --entry-point=io.quarkus.funqy.gcp.functions.FunqyBackgroundFunction \
+  --trigger-resource hello_topic --trigger-event google.pubsub.topic.publish \
+  --runtime=java11 --source=deployment
+```
+
+You can then invoke your function via `gcloud`:
+
+```shell script
+gcloud functions call quarkus-example-pubsub --data '{"data":"HelloWorld"}'
+```
+
+### Storage event
+
+To deploy a background function that listen to Storage event, you can use the following `gcloud` command:
+
+```shell script
+gcloud beta functions deploy quarkus-funqy-storage --entry-point=io.quarkus.funqy.gcp.functions.FunqyBackgroundFunction \
```

Comment:
For lambda, the extension generates a helper script within target/ than can create, update, delete, and invoke the lambda.  If you have time to add that, great, if not, its an issue we can post and do it later.

Reply:
I'll create an issue for this to implement it for both Funqy and HTTP binding.
As I didn't want to miss the 1.6 target, I prefere to build our support step by step instead of providing full functionalities.
Maybe I should set the extension status to experimental waiting to have all the functionalities (easy testing is also missing)